example/user/user.pb.gorm.go

```diff
@@ -3,6 +3,7 @@ package user
 import (
 	context "context"
 	fmt "fmt"
+	timestamp "github.com/golang/protobuf/ptypes/timestamp"
```

Comment:
the same thing, it should not use old proto bindings timestamppb "google.golang.org/protobuf/types/known/timestamppb" should be used instead